File: C program/New Batch/Pattern1.cpp
// * * * * 
// * * * * 
// * * * * 
// * * * * 
#include<stdio.h>
int main(){
    int num;
    printf("Enter a number: ");
    scanf("%d",&num);
    for(int i=0;i<num;i++){
        for(int j=0;j<num;j++){
            printf("* ");
        }
        printf("\n");
    }
    return 0;
}

File: Java Codes/New Batch/Pattern1.java
import java.util.Scanner;
class Pattern1{
	public static void main(String[] args) {
		int n;
		System.out.print("Enter a number: ");
		Scanner sc = new Scanner(System.in);
		n = sc.nextInt();
		for(int i=0;i<n;i++){
			for(int j=0;j<i+1;j++){
				System.out.print("* ");
			}
			System.out.println();
		}
	}
}